CouchbaseLite.xcodeproj/project.pbxproj

```diff
@@ -5565,6 +5567,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 27BE3B451E4D63AF0012B74A /* CBL Swift.xcconfig */;
 			buildSettings = {
+				BUILD_LIBRARY_FOR_DISTRIBUTION = YES;
```

Comment:
Do you know what is this config? Should it go to the xcconfig file?

Reply:
This is necessary when we make it xcframework

Reply:
At least, it needs to be in xcconfig file, not in the xcode project. The following should be find out (maybe look to see how other libraries are doing).
1. Should BUILD_LIBRARY_FOR_DISTRIBUTION be enabled on both Swift and Objective-C target?
2. Should BUILD_LIBRARY_FOR_DISTRIBUTION be enabled on both debug and release build.
3. Should we just enable it at the xcodebuild command?

Reply:
as per the understanding this will create the `<arch-type>.swiftinterface` file for swift target. which will be used to compatible with future swift compiler versions.

Reply:
I am getting an error called "No 'swiftinterface' files found within <path to module map>" when using as a command line argument in archive.